jquery.flexslider.js

```diff
@@ -1032,6 +1035,25 @@
       slider.vars.removed(slider);
     };
 
+	slider.destroy = function() {
```

Comment:
There is a little more to add to this destroy function (and other places in the code). There are several calls to setTimeout and setInterval which need to be canceled when the timer is destoyed. The main bulk of code needs variables adding to store the timers and then the destroy function needs to call clearTimerout(<<var>>) and clearTimerout(<<var>>) for each one.